src/Simplex/Messaging/Agent.hs

```diff
@@ -1482,6 +1602,100 @@ processSMPTransmission c@AgentClient {smpClients, subQ} (srv, v, sessId, rId, cm
                 connectReplyQueues c cData ownConnInfo smpQueues `catchError` (notify . ERR)
               _ -> prohibited
 
+        -- processed by queue sender
+        qAddMsg :: L.NonEmpty (SMPQueueUri, Maybe SndQAddr) -> m ()
+        qAddMsg ((_, Nothing) :| _) = throwError $ INTERNAL "adding queue without switching is not supported"
```

Comment:
```suggestion
        qAddMsg ((_, Nothing) :| _) = throwError $ INTERNAL "QADD: adding queue without switching is not supported"
```

Reply:
possibly add constructor to Agent error for switch errors (A_SWITCH)

Reply:
possibly add should check whether some other queue in connection already rotates the main queue, and then the old one should be deleted